src/src/models/SelectModel.ts

```diff
@@ -0,0 +1,19 @@
+import OptionModel from "./OptionModel";
+
+export interface SelectModel {
+    error: string;
+    disabled: boolean;
+    mandatory: boolean;
+    value: string;
+    tabindex: number;
+    fieldName: string;
+    fieldIcon: string;
+    multiple: boolean;
+    customClass: string;
+    hasValidation: boolean;
+    isValid: boolean;
+    showLabel: boolean;
+    showDefault: boolean;
+    defaultLabel: string;
+    options: OptionModel[];
+};
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
}
```